src/scripts/import-projects.ts

```diff
@@ -36,25 +39,55 @@ async function skipTargetIfFoundInLog(
     return false;
   }
 }
+
+async function parseLog(logFile: string): Promise<string[]> {
+  return new Promise((resolve, reject) => {
+    const importedTargets: string[] = [];
+    fs.createReadStream(logFile)
+      .pipe(split())
+      .on('data', (lineObj) => {
+        if (!lineObj) {
+          return;
+        }
+        try {
+          const data = JSON.parse(lineObj);
+          const { orgId, integrationId, target } = data;
+          importedTargets.push(
+            generateImportedTargetData(orgId, integrationId, target),
```

Comment:
generate the targetID to be a string using the same function that writes the ID into the log when target was imported, so we can use string[] comparisons